subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy

```diff
@@ -55,4 +66,174 @@ class CheckstylePluginIntegrationTest extends WellBehavedPluginTest {
         then:
         executedAndNotSkipped ':checkstyleMain'
     }
+
+    // region: Enable External DTDs
```

Comment:
The flag has been around for 3.5 years, looks like.  Since Checkstyle 8.18.  https://github.com/checkstyle/checkstyle/commit/180b4fe37a2249d4489d584505f2b7b3ab162ec6

I think that's long enough there's no real benefit to testing with older versions.